stories/header.stories.js

```diff
@@ -1,39 +1,66 @@
+/**
+ * Header stories.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 /**
  * External dependencies
  */
 import { storiesOf } from '@storybook/react';
+
 /**
  * Internal dependencies
  */
 import Header from '../assets/js/components/Header';
-import { STORE_NAME as CORE_SITE } from '../assets/js/googlesitekit/datastore/site/constants';
-import { STORE_NAME as CORE_USER } from '../assets/js/googlesitekit/datastore/user/constants';
-import { WithTestRegistry } from '../tests/js/utils';
+import DateRangeSelector from '../assets/js/components/DateRangeSelector';
+import { createTestRegistry, provideSiteInfo, provideUserAuthentication, WithTestRegistry } from '../tests/js/utils';
+
+function Setup( props ) {
+	return (
+		<WithTestRegistry { ...props }>
+			<Header>
+				{ props.children }
+			</Header>
+		</WithTestRegistry>
```

Comment:
Let's not use this abstraction since we want to see how the `Header` component is used in each story and it isn't worth extracting a component just for the test registry so please inline below.